.github/CODEOWNERS

```diff
@@ -29,23 +29,22 @@
 # directory at the root of the repository and any of its
 # subdirectories.
 #/build/logs/ @doctocat
-/tests/disaster-recovery/regional-dr/ @red-hat-storage/turquoise-squad
+/tests/functional/disaster-recovery/regional-dr/ @red-hat-storage/turquoise-squad
 /tests/lvmo/ @red-hat-storage/aqua-squad
-/tests/manage/z_cluster/ @red-hat-storage/brown-squad
-/tests/manage/pv_services/ @red-hat-storage/green-squad
-/tests/manage/storageclass/ @red-hat-storage/green-squad
-/tests/manage/monitoring/ @red-hat-storage/blue-squad
-/tests/manage/mcg/ @red-hat-storage/red-squad
-/tests/manage/rgw/ @red-hat-storage/red-squad
-/tests/ecosystem/upgrade/ @red-hat-storage/purple-squad
-/tests/e2e/flowtest/ @red-hat-storage/magenta-squad
-/tests/e2e/lifecycle/ @red-hat-storage/magenta-squad
-/tests/e2e/workloads/ @red-hat-storage/magenta-squad
-/tests/e2e/performance/ @red-hat-storage/grey-squad
-/tests/e2e/scale/ @red-hat-storage/orange-squad
-/tests/encryption @red-hat-storage/green-squad
-/tests/ui/ @red-hat-storage/black-squad
-/ocs_ci/ocs/ui/ @red-hat-storage/black-squad
+/tests/functional/z_cluster/ @red-hat-storage/brown-squad
+/tests/functional/pv/pv_services/ @red-hat-storage/green-squad
+/tests/functional/storageclass/ @red-hat-storage/green-squad
+/tests/functional/encryption/ @red-hat-storage/green-squad
+/tests/functional/monitoring/ @red-hat-storage/blue-squad
+/tests/functional/object/mcg/  @red-hat-storage/red-squad
+/tests/functional/object/rgw/  @red-hat-storage/red-squad
+/tests/functional/upgrade/ @red-hat-storage/purple-squad
+/tests/cross_functional/flowtest/ @red-hat-storage/magenta-squad
+/tests/functional/workloads/ @red-hat-storage/magenta-squad
+/tests/cross_functional/performance/ @red-hat-storage/grey-squad
+/tests/cross_functional/scale/ @red-hat-storage/orange-squad
+/tests/functional/ui/ @red-hat-storage/black-squad
+/tests/cross_functional/ui/ @red-hat-storage/black-squad
 
```

Comment:
please, return back `/ocs_ci/ocs/ui/ @red-hat-storage/black-squad`
thank you for your effort on this!

Reply:
done 👍